packages/main/src/themes/base/Input-parameters.css

```diff
@@ -87,8 +87,8 @@
 	--_ui5_input_information_icon_padding: .1875rem .375rem .1875rem .4375rem;
 	--_ui5_input_information_focused_icon_padding: .1875rem .375rem .1875rem .4375rem;
 	--_ui5_input_custom_icon_padding: .25rem .5rem;
-	--_ui5_input_error_warning_custom_icon_padding: .1875rem .5rem;
-	--_ui5_input_error_warning_custom_focused_icon_padding: .1875rem .5rem;
+	--_ui5_input_information_error_warning_custom_icon_padding: .1875rem .5rem;
+	--_ui5_input_information_error_warning_custom_focused_icon_padding: .1875rem .5rem;
 	--_ui5_input_information_custom_icon_padding: .1875rem .5rem;
```

Comment:
_ui5_input_information_custom_icon_padding has different values from the error and warning ones in high contrast white/black Fiori 3 themes. 

Also as it is, both `--_ui5_input_information_custom_icon_padding` and --`_ui5_input_information_custom_focused_icon_padding` params are not used anywhere